src/sprites/rendered-target.js

```diff
@@ -458,6 +469,17 @@ class RenderedTarget extends Target {
         this.runtime.requestTargetsUpdate(this);
     }
 
+    /**
+     * Add a sound, taking care to avoid duplicate names.
+     * @param {!object} soundObject Object representing the sound.
+     * @param {!int} index Index at which to add costume
+     */
+    addSoundAt (soundObject, index) {
```

Comment:
instead of a new method that duplicates some of the code in `addSound`, could we just add an optional param to addSound that lets you specify the index, defaulting to adding the sound to the end? It would reduce the duplication and lower the surface are for bugs. (ditto for costumes methods)